src/com/epam/javatrainings/annotations/validation/BoundaryAnnotationProcessor.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.javatrainings.annotations.validation;
+
+import java.lang.annotation.Annotation;
+
+public abstract class BoundaryAnnotationProcessor extends AnnotationProcessor {
+    abstract String validate(Annotation annotation, Object value);
```

Comment:
why we need this abstract method here if we have the same method in the parent ?